lib/msal-node/src/cache/NodeCacheManager.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require('fs');
+
+export class NodeCacheManager {
+    cache: string;
+
+    /**
+     * Read contents of the cache blob to in memoryCache
+     * @param cachePath
+     */
+    readFromFile(cacheLocation: string): string {
+        this.cache = fs.readFileSync(cacheLocation, 'utf8');
```

Comment:
Can you make this async? https://nodejs.org/api/fs.html#fs_fs_promises_api

Reply:
Done